### PR TITLE
Adds a sponsor splash screen before the scenario lobby

### PR DESCRIPTION
**Pull request by andrewmunroe**

### CODE REVIEW

File: frontend/lobby/splash.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as movember from './images/funded-by-movember.png';
+
+// Include our SCSS (via webpack magic)
+import './lobby.scss';
```
Comment (bradenmacdonald) [inline frontend/lobby/splash.tsx]: I think we only need this in one file, so the existing line in `lobby.tsx` should be fine.

**Comment by bradenmacdonald** [comment]
FYI @andrewmunroe  this is showing up as distorted on my computer for some reason:

screen shot 2018-04-19 at 3 41 20 pm

**Comment by andrewmunroe** [comment]
Weird @bradenmacdonald - it looks good in Firefox but is distorted in chrome. Adding `width: 100%` and `height: 100%` to `.splash > div img` in `lobby.scss` seems to fix the problem.

**Comment by bradenmacdonald** [comment]
@andrewmunroe Thanks! Fixed :)

**Pull request merged**